src/transforms/posthtml.js

```diff
@@ -1,7 +1,17 @@
 const loadPlugins = require('../utils/loadPlugins');
 const posthtml = require('posthtml');
+const posthtmlParse = require('posthtml-parser');
 
-module.exports = async function(asset) {
+async function parse(code, asset) {
+  var config = await getConfig(asset);
```

Comment:
Is the configuration format identical in parser and transform?

Reply:
Yes, but different options are used. Do you prefer these configs to be different objects?

Reply:
As long as they can’t conflict it should be fine but it sounds like they can

Reply:
How about putting these options into a new htmlparser2rc ?

Reply:
I was thinking more of something like this:

Reply:
But as you said they won't conflict I think it's safe for now to just let this PR be as is

Reply:
I think this will result in getting the config twice... once for parse, and once for transform. Maybe ok because of the cache? Just pointing that out.